src/lib.rs

```diff
@@ -69,9 +69,9 @@ pub fn assert_contents<P: AsRef<Path>>(path: P, actual: &str) {
         let expected = dos2unix(&expected_s);
 
         if expected != actual {
-            let changeset =
-                Changeset::new(expected.as_ref(), actual.as_ref(), "\n");
-            println!("{}", changeset);
+            let diff =
+                unified_diff(Algorithm::Myers, &expected, &actual, 5, None);
```

Comment:
Myers is the default algorithm in git, so that seems fine. 5 is the number of lines before and after. `None` could be replaced with a header representing the two file names, not sure if we care.

Reply:
Yeah there's only one file and it should be clear from context so agreed that it's not useful to add a header